<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>15</width>
    <height>15</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot_ecPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>CMD Pannel</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ecCMD1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ecCMD2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ecCMD3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ecCMD4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ecSendProcessData">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_debug_clear">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>CLC</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Pannel</string>
    </property>
    <widget class="QCheckBox" name="checkBox_isPrintSDO">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Print SDO</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_isPrintMAP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Print MAP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_isPrintToConsole">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Print to Console</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ecRun">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ecStop">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_QR_display">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>QR</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>CHs</string>
    </property>
    <widget class="QCheckBox" name="checkBox_CH1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CH1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_CH2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CH2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_CH3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CH3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_CH4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CH4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_PlotAutoZoom">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>AutoZoom</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QComboBox" name="comboBox_ecadaptert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ecScan">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ecadaptert_dec">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_displayDebug">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>hello word</string>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFind">
    <property name="title">
     <string>Adaptert</string>
    </property>
    <addaction name="action_ScanAdapyert"/>
    <addaction name="action_Scan_Slaves"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QMenu" name="menuPlot_setting">
     <property name="title">
      <string>Plot</string>
     </property>
     <addaction name="action_PlotAdapt_zoom"/>
     <addaction name="separator"/>
     <addaction name="action_Adapt_zoom1"/>
     <addaction name="action_Adapt_zoom2"/>
     <addaction name="action_Adapt_zoom3"/>
     <addaction name="action_Adapt_zoom4"/>
    </widget>
    <widget class="QMenu" name="menuWindows">
     <property name="title">
      <string>Windows</string>
     </property>
     <addaction name="action_windowsMax"/>
     <addaction name="action_windowsRestore"/>
     <addaction name="action_WindowsNoneTitle"/>
    </widget>
    <addaction name="menuWindows"/>
    <addaction name="menuPlot_setting"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_ecRun"/>
    <addaction name="action_ecStop"/>
    <addaction name="action_ecSend"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFind"/>
   <addaction name="menuTools"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_ScanAdapyert">
   <property name="text">
    <string>Scan Adaptert</string>
   </property>
  </action>
  <action name="action_PlotAdapt_zoom">
   <property name="text">
    <string>Adapt-zoom-default</string>
   </property>
  </action>
  <action name="action_Scan_Slaves">
   <property name="text">
    <string>Scan Slaves</string>
   </property>
  </action>
  <action name="action_ecRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="action_ecStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="action_ecSend">
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="action_Adapt_zoom1">
   <property name="text">
    <string>Adapt-zoom1</string>
   </property>
  </action>
  <action name="action_Adapt_zoom2">
   <property name="text">
    <string>Adapt-zoom2</string>
   </property>
  </action>
  <action name="action_Adapt_zoom3">
   <property name="text">
    <string>Adapt-zoom3</string>
   </property>
  </action>
  <action name="action_windowsMax">
   <property name="text">
    <string>Maximize</string>
   </property>
  </action>
  <action name="action_windowsRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_WindowsNoneTitle">
   <property name="text">
    <string>NoneTitle</string>
   </property>
  </action>
  <action name="action_Adapt_zoom4">
   <property name="text">
    <string>Adapt-zoom4</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
